### ISSUE TITLE
Trezor wallet does not appear to be syncing on UI but it actually is

### ISSUE THREAD

**Issue by bitcoin-solutions** [bitcoin-solutions/multibit-hd#334]
With a trezor plugged in I have seen on the UI that it does not appear to be syncing but from my Mac network monitor I see that it is. Then suddenly it is 100% synced.

Looks like something is not being hooked up properly and events aren't propagating properly.

**Comment by bitcoin-solutions** [comment]
Also seeing this on Repair Wallet for a Trezor wallet which might be easier to debug

**Comment by gary-rowe** [comment]
Yep, I've seen it here too so good that we have an issue to cover it.

**Comment by bitcoin-solutions** [comment]
Pushed some changes to MultiBitPeerEventListener to make it slightly more 'shouty'.
Keep on eye on whether you still see this issue or not over the next few days.

**Comment by bitcoin-solutions** [comment]
I have pushed more fixes so that:
+ MultiBitPeerEventListener only 'shouts' on 1% increments of block downloads, rather than every block as it was doing before. (This reduces an event cascade on syncing)
+ Restore wallet and repair wallet use the BitcoinNetworkStatus enum directly to work out status rather than the block count.

I think I've fixed this one now as I haven't seen it in my testing so awaiting review and closing.
